corehq/apps/api/odata/serializers.py

```diff
@@ -77,7 +83,7 @@ def serialize_documents_using_config(documents, config, table_id):
         for row_number, document in enumerate(documents):
             rows = table.get_rows(
                 document,
-                row_number,
+                row_number + (self.offset * self.limit),
```

Comment:
nit: might be clearer if this was a property/method. e.g. `self.get_full_row_number()` or something. or even if `(self.offset * self.limit)` was pulled out into `self.start_doc_count` or something...